src/main/java/at/ac/tuwien/kr/alpha/solver/heuristics/BerkMin.java

```diff
@@ -128,22 +132,20 @@ public void newNoGood(NoGood newNoGood) {
 
 	@Override
 	public void newNoGoods(Collection<NoGood> newNoGoods) {
-		for (NoGood noGood : newNoGoods) {
-			newNoGood(noGood);
-		}
+		newNoGoods.forEach(this::newNoGood);
```

Comment:
I saw that you are not hostile to using Java 8, so I did that here. Happy to revert this.